lib/runtime-c-api/tests/test-import-function.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include "../wasmer.h"
+#include <assert.h>
+#include <stdint.h>
+
+static print_str_called = false;
+
+void print_str(int32_t ptr, int32_t len, wasmer_instance_context_t *ctx) {
+    printf("In print_str\n");
+    print_str_called = true;
+}
+
+int main()
+{
+    wasmer_import_object_t *import_object = wasmer_import_object_new();
+    wasmer_value_tag params_sig[] = {WASM_I32, WASM_I32};
+    wasmer_value_tag returns_sig[] = {};
+    wasmer_imports_set_import_func(import_object, "env", "print_str", print_str, params_sig, 2, returns_sig, 0);
```

Comment:
Could be possible to have a macro for this? (regarding params and returns signature)

Reply:
I don't think a macro is necessary for this. Using simple arrays here is pretty simple.

Reply:
I also had some ideas about making the API more user friendly. Effectively sugar coating 😛 

I think keeping the C API simple is a good start, and letting other libraries wrap and make it easier. A higher level API for C++ may be nice, but it depends on how much we want to support.

It's also probably more idiomatic C to not use macros in a library like this.